### PR TITLE
Dockerface

### PR DESCRIPTION
**Pull request by natanielruiz**
**[Insert the URL to be listed]**
https://github.com/natanielruiz/dockerface

**[Explain what the project is about and why it should be listed]**
Dockerface is an easy to install and use deep learning face detection for images and video in a docker container. It is meant for research and open for industry and personal use. It deploys a trained Faster R-CNN network on Caffe through an easy to use docker image. Bring your videos and images, run dockerface and obtain videos and images with bounding boxes of face detections and an easy to use face detection annotation text file.

I also created a new section called Machine Learning. Docker is widely used in the field so it might make more people submit this style of applications to the list.

Cheers!

Nataniel

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- The project submitted is conform to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.

**RULES (because you probably didn't read the guidelines):**

Links will be tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot). Please make sure your URL to be listed is unique (do some search queries), otherwise put a link at the end of `README.md` file.
 
- FORMAT
  - Use the following format: `[Name](Link) - Description.`
  - `[Name]` should be the title of the project
  - `(Link)` should be pointing to a GitHub repository, not to a website
  - `Description.` should end with a full stop/period
  - For GitHub repositories, add the name of the author with a link to its GitHub account after the description, like `By [@author](GitHub account)`
- ORDER: The entries are in lexical order (English letter order) 
- LABEL (Optional)
  - $$$ indicates if one needs a paying subscription to use the project.
  <br>Add $$$ next to the project name like `[Name $$$](Link)`
  - BETA indicates that the project is in beta phase.
  <br>Add BETA next to the project name like `[Name BETA](Link)`

### CODE REVIEW

**Comment by veggiemonk** [comment]
Hi @natanielruiz,

We are in the middle of refactoring the list and all the sections. So we have to wait a little until this PR is merged. 
We could include a new section but Machine Learning isn't really related to docker per say. We could however start a new section for heavy processing app such as those project which use nvidia CUDA docker images in it. Is that something that would be OK with you?
Could you help us make that section happen ? If you are interested, you could even be a maintainer of that list since I don't who which of us has experience with those technologies. Although they are really interesting. 

What do you think?

**Comment by natanielruiz** [comment]
Hey there @veggiemonk,

I agree that an NVIDIA CUDA Docker section would be the best thing. I don't know of many other applications that are similar to Dockerface but I know that frameworks such as torch, pytorch, etc. are deployed on docker images. I can help you if you want sure.

Cheers

**Comment by veggiemonk** [comment]
Hi @natanielruiz 
Maybe Dockerface would have more interest in https://github.com/josephmisiti/awesome-machine-learning. If you can make a pull request with a section containing some high quality projects (with documentation on how to use it, how to install it and examples) you can be a maintainer for sure!
What do you say?

**Comment by veggiemonk** [comment]
Ok this is abandoned
